src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -253,7 +253,7 @@ private void InitializeCore()
             DefineFullCategoryNames(LibraryRootCategories, "");
             InsertClassesIntoTree(LibraryRootCategories);
 
-            ChangeRootCategoryExpandState(BuiltinNodeCategories.GEOMETRY, true);
+            ChangeRootCategoryExpandState(BuiltinNodeCategories.GEOMETRY_CATEGORY, true);
```

Comment:
I'm not sure how does this change anything, perhaps I'm missing something: the old `GEOMETRY` has the value of `Geometry`, and the new `GEOMETRY_CATEGORY` has the same value `Geometry`.